src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -21,6 +20,7 @@
 using System.Windows.Controls.Primitives;
 using System.Windows.Documents;
 using System.Windows.Input;
+using AccessibilityInsights.SharedUx.Enums;
```

Comment:
order?

Reply:
I will make a separate pass through some of these files to fix the order all at once